docs/development/contributing-to-docs.md

```diff
@@ -0,0 +1,139 @@
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+Druid is a [community-led project](https://druid.apache.org/community/) and we are delighted to receive contributions of anything from minor fixes to docs to big new features.
+
+Druid docs contributors:
+
+* Improve existing content
+* Create new content
+
+## Getting started
+
+Druid docs contributors can open an issue about documentation, or contribute a change with a pull request (PR).
+
+The open source Druid docs are located here:
+https://druid.apache.org/docs/latest/design/index.html
+
+Some of the Druid docs are incorporated into the Imply docs:
+https://docs.imply.io/latest/apache-druid-doc/
+
+If you need to update a Druid doc, locate and update the doc in the Druid repo, following the instructions below. Once a month, we run a script to update the Imply docs repo with recent updates to the Druid docs.
+
+## Druid repo branches
+
+The Druid team works on Apache Druid master, and then branches to 0.17, 0.18, etc and 017-iap (which stands for Imply Analytics Platform).
+
+See [CONTRIBUTING.md](https://github.com/apache/incubator-druid/blob/master/CONTRIBUTING.md) for instructions on contributing to Apache Druid.
+
+## Before you begin
+
+Before you can contribute to the Druid docs for the first time, you must complete the following steps:
+
+  1. Fork the [Druid repo](https://github.com/apache/druid). Your fork will be the ```origin``` remote.
+  2. Clone the Druid repo from your fork.
+  3. Set up your remotes locally ```upstream``` in the Druid repo in ```.git/config```:
+  ~~~~
+  [remote "upstream"]
+  	url = https://github.com/apache/druid.git
+  	fetch = +refs/heads/*:refs/remotes/upstream/*
+  	pushurl = no_push
+  [branch "master"]
+  	remote = upstream
+  	merge = refs/heads/master
+  [remote "origin"]
+  		url = https://github.com/{my-git-id}/druid.git
+  		fetch = +refs/heads/*:refs/remotes/upstream/*
+  [branch "master"]
+  		remote = origin
+  		merge = refs/heads/master
+  ~~~~
+
+  For ```upstream```, ```push_url = no_push``` means you won’t accidentally push to upstream.
+  Make sure to put your github id for {my-git-id}.
+  4. ```git config --list --show-origin``` to make sure you’ve got your email configured. If you need to set your email, you can set it per repo or globally. Global instructions [here](https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git).
+  5. Docusaurus?
```

Comment:
Since this is in before you start, I think probably

```suggestion
  5. Install Docusaurus so that you can build the site locally. Run either `npm install` or `yarn install` in the `website` directory.
```